Gems/LmbrCentral/Code/Source/Scripting/EditorTagComponent.cpp

```diff
@@ -113,15 +113,15 @@ namespace LmbrCentral
 
         TagComponentNotificationsBus::Event(entityId, &TagComponentNotificationsBus::Events::OnTagAdded, tag);
         TagGlobalNotificationBus::Event(tag, &TagGlobalNotificationBus::Events::OnEntityTagAdded, entityId);
-        TagGlobalRequestBus::MultiHandler::BusConnect(tag);
+        // Intentionally don't connect to the TagGlobalRequestBus for the editor component because its bus Id and params are
+        // not tied to any entity Id. This can collide with the runtime tag component which will have the same tag name
```

Comment:
It feels weird to add a comment about a bus that is not being connected to. I recommend that you remove this since the reasoning can be tracked via this PR and commit history.

Reply:
Actually this is subtle. The comment is probably a really, really good idea.